frontend/app/routes/$lang+/_public+/status+/index.tsx

```diff
@@ -97,14 +97,13 @@ export async function action({ context: { session }, params, request }: ActionFu
   const lookupService = getLookupService();
   const { sin, code } = parsedDataResult.data;
   const statusId = await applicationStatusService.getStatusId(sin, code);
-
   const clientStatusList = await lookupService.getAllClientFriendlyStatuses();
   const clientFriendlyStatus = clientStatusList.find((status) => status.id === statusId);
 
   return json({
     status: {
       ...(clientFriendlyStatus ?? {}),
-      alertType: clientFriendlyStatus?.id === CLIENT_STATUS_SUCCESS_ID ? 'success' : 'info',
+      alertType: statusId ? (clientFriendlyStatus?.id === CLIENT_STATUS_SUCCESS_ID ? 'success' : 'info') : 'warning',
```

Comment:
**Nitpick:** Making the following change will display the same icon that the Power Platform Status Checker version shows.

```suggestion
      alertType: statusId ? (clientFriendlyStatus?.id === CLIENT_STATUS_SUCCESS_ID ? 'success' : 'info') : 'danger',
```
**Warning:**
![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/55502055/9fd85fc1-68dd-49bd-a569-f8fa50a02da2)

**Danger:**
![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/55502055/1b7204de-03d1-4cfc-bad0-f2143117cef6)